audmath/core/api.py

```diff
@@ -193,6 +194,86 @@ def inverse_normal_distribution(
     return _force_float(x)
 
 
+def rms(
+        x: typing.Union[float, typing.Sequence[float], np.ndarray],
+) -> float:
```

Comment:
It is returning `np.float64`, should I be more explicit here or stay with `float`?

Reply:
I didn't check, but I would expect it returns `np.float32` if the in the dtype of the array is also `np.float32`.

Reply:
```
x = np.zeros((10), dtype=np.float32)
type(audmath.rms(x))
```
```
<class 'numpy.float32'>
```

So the question is if we should return `float(rms(...))` instead?

Reply:
I changed return type to `typing.Union[float, np.ndarray]`

Reply:
If we want to be slightly more precise we could replace `float` with `np.floating`:

![image](https://user-images.githubusercontent.com/10383417/203794004-1a723d09-b0fd-4482-89eb-4eb05ff811ad.png)

But then we should probably return `np.float64(0)` in the empty signal case.

Reply:
Good idea, I changed the return type to include `np.floating`. I also return `np.float64(0)` to be consistent with what we return for single input values.

Theoretically, I'm not sure if it would still be ok with `float`, because there is a slight inconsitency.
On the one hand we have:
```python
>>> isinstance(0., np.floating)
False
```
But if we use the [canonical way to check](https://github.com/numpy/numpy/issues/17325#issuecomment-693297036) we get:
```python
>>> np.issubdtype(type(0.), np.floating)
True
```

Reply:
Ok, strange.

Another thing I just realized - we have a `_force_float()` function that we use with the result value of `inverse_normal_distribution()` to explicitly convert to standard `float` if the result is a scalar. I wonder if we should be consistent across functions, though I'm more in favor of always returning `np.float64` as we do here now. What do you think?

Reply:
I agree, I created #13 to update the return type in `audmath.inverse_normal_distribution()` as this shouldn't have bad consequences.